fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -213,24 +203,24 @@ protected RdfStream getResourceTriples() {
 
         final Predicate<Triple> tripleFilter;
         if (ldpPreferences.prefersServerManaged()) {
-            tripleFilter = alwaysTrue();
+            tripleFilter = x -> true;
         } else {
-            tripleFilter = and(not(ManagedRdf.isManagedTriple), not(new Predicate<Triple>() {
+            tripleFilter = new Predicate<Triple>() {
                 @Override
-                public boolean apply(final Triple input) {
+                public boolean test(final Triple input) {
                     return input.getPredicate().equals(RDF.type.asNode())
                             && isManagedNamespace.apply(input.getObject().getNameSpace());
                 }
-            }));
+            }.negate().and(not(isManagedTriple)::apply);
```

Comment:
This whole mess of "isManagedFoot" and "isManagedEar" and specific type-tests needs to have the stuffing kicked out of it and beaten into something that makes some kind of sense.